src/models/propulsion/FGRocket.h

```diff
@@ -41,6 +41,7 @@ INCLUDES
 #include "FGEngine.h"
 #include "math/FGTable.h"
 #include "math/FGFunction.h"
+#include "RocketState.h"
```

Comment:
Mind calling it like below, so that it's easier to see the custom classes that we've added?
```suggestion
#include "DARocketState.h"
```